Framework/TestWorkflows/src/o2DiamondWorkflowLeaky.cxx

```diff
@@ -47,7 +47,7 @@ void customize(std::vector<SendingPolicy>& policies)
 {
   policies.push_back(SendingPolicy{
     .matcher = DeviceMatchers::matchByName("A"),
-    .send = [](FairMQDeviceProxy& proxy, fair::mq::Parts& parts, ChannelIndex channelIndex) {
+    .send = [](FairMQDeviceProxy& proxy, fair::mq::Parts& parts, ChannelIndex channelIndex, ServiceRegistry& registry) {
```

Comment:
Could you have a separate commit / PR where you simply modify the API of the SendingPolicy to take the `ServiceRegistry&`? This we should be able to do and merge immediately.